scriptworker/artifacts.py

```diff
@@ -424,3 +429,22 @@ def get_optional_artifacts_per_task_id(upstream_artifacts):
             )
 
     return optional_artifacts_per_task_id
+
+
+def assert_is_parent(path, parent_dir, exception=ScriptWorkerTaskException):
```

Comment:
What's the use case for passing down `exception`? I don't see it used anywhere in this PR. I wonder if it's case of YAGNI.

Reply:
Hm. I think I was angling for raising a `CoTError` if we hit this during `cot.verify`, and a `ScriptWorkerTaskException` elsewhere, but it looks like I don't use it.